src/ProductOptionListBuilder.php

```diff
@@ -0,0 +1,29 @@
+<?php
+
+namespace Drupal\commerce_option;
```

Comment:
As per the updated standards, we should be moving the List Builder and any other files that are under the /src folder to a proper directory. For example, for this file, it should be moved to under, `/src/Entity/ListBuilder`. But I know that we are porting this entity exactly as the Commerce Attribute so, let's leave it as is for now.

But in the future, it is something to keep in mind so that we don't have a lot of loose files hanging around under the /src folder. It makes things a bit more cleaner and organized.

Reply:
Didn't have to but thanks for changing that.